ui/desktop/playwright.config.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+/* eslint-disable no-undef */
+
+/** @type {import('@playwright/test').PlaywrightTestConfig} */
+const config = {
+  outputDir: './tests/e2e/artifacts',
```

Comment:
what is stored here?

Reply:
Any thing we save from tests, usually traces